taprpc/tapcommon.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package taprpc;
+
+option go_package = "github.com/lightninglabs/taproot-assets/taprpc";
+
+// Represents a Bitcoin transaction outpoint.
+message OutPoint {
+    /*
+    Raw bytes representing the transaction id.
+    */
+    bytes txid = 1;
```

Comment:
for the txid specifically, should we use string here?

Reply:
or i guess this is a refactor of a message that's already in use

Reply:
Yeah, it's an existing message, don't think we should refactor it in this PR.